libraries/abstractions/pkcs11/utest/CMakeLists.txt

```diff
@@ -0,0 +1,142 @@
+project ("abstractions pkcs11 cmock unit test")
+cmake_minimum_required (VERSION 3.13)
+set(3rdparty_dir "${AFR_ROOT_DIR}/libraries/3rdparty")
+set(abstractions_dir "${AFR_ROOT_DIR}/libraries/abstractions")
+set(standard_dir "${AFR_ROOT_DIR}/libraries/freertos_plus/standard")
+set(kernel_dir "${AFR_ROOT_DIR}/freertos_kernel")
+set(common_dir "${AFR_ROOT_DIR}/libraries/c_sdk/standard/common")
+
+# Mock library
+list(APPEND mock_list
+    "${abstractions_dir}/pkcs11/include/iot_pkcs11_pal.h"
+    "${kernel_dir}/include/portable.h"
+    "${kernel_dir}/include/queue.h"
+    "${common_dir}/include/iot_logging_task.h"
+    "${standard_dir}/crypto/include/iot_crypto.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/platform_util.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/ctr_drbg.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/sha256.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/base64.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/bignum.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/entropy.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/md.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/rsa.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/ecp.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/ecdsa.h"
+    "${3rdparty_dir}/mbedtls_utils/mbedtls_error.h"
+    "${3rdparty_dir}/mbedtls/include/mbedtls/pk.h"
+    "${standard_dir}/utils/include/iot_pki_utils.h"
+    )
+
+create_mock_list(pkcs11_mbedtls_mock 
+                 "${mock_list}"
+                 "${abstractions_dir}/pkcs11/utest/project.yml"
+                 )
+
+target_compile_definitions(pkcs11_mbedtls_mock PUBLIC
+        portHAS_STACK_OVERFLOW_CHECKING=1
+        portUSING_MPU_WRAPPERS=1
+        MPU_WRAPPERS_INCLUDED_FROM_API_FILE
+        )
+
+target_include_directories(pkcs11_mbedtls_mock PUBLIC
+        "${standard_dir}/crypto/include"
+        "${common_dir}/include"
+        "${3rdparty_dir}/mbedtls/include/mbedtls"
+        "${3rdparty_dir}/mbedtls_utils"
+        "${kernel_dir}/include"
+        "${standard_dir}/utils/include"
+        "${AFR_ROOT_DIR}/tests/unit_test/linux/config_files"
+        "${abstractions_dir}/pkcs11/include"
+        "${standard_dir}/pkcs11/include"
+        )
+
+# PKCS #11 header files must always come before the mbedTLS header files
+target_include_directories(pkcs11_mbedtls_mock BEFORE PUBLIC
+        "${3rdparty_dir}/pkcs11"
+        )
+
+add_dependencies(pkcs11_mbedtls_mock
+                 cmock
+                 unity
+                 )
+
+# Real libraries
+add_library(pkcs11_mbedtls_real STATIC
+        ${3rdparty_dir}/mbedtls/library/pk_wrap.c
```

Comment:
For some reason I can't find that file "pk_wrap.c"
compilation fails on my machine